packages/auto-approve/test/check-pr.test.ts

```diff
@@ -181,16 +199,73 @@ describe('check pr against config', async () => {
           resolve(fixturesPath, 'config', 'valid-schemas', 'valid-schema2.yml'),
           'utf8'
         )
-      ) as {rules: ValidPr[]};
+      ) as {rules: checkPR.ValidPr[]};
 
       const pr = require(resolve(
         fixturesPath,
         'events',
         'pull_request_opened_right_author_and_title_partial_schema'
       ));
 
-      const prMatchesConfig = await checkPRAgainstConfig(validPR, pr, octokit);
+      const scopes = listChangedFilesPR(200, [
+        {filename: 'README.md', sha: '1234'},
+        {filename: '.github/readme/synth.metadata/synth.metadata', sha: '1234'},
+      ]);
+
+      const prMatchesConfig = await checkPR.checkPRAgainstConfig(
+        validPR,
+        pr,
+        octokit
+      );
+      scopes.done();
+      assert.ok(prMatchesConfig);
+    });
+  });
+
+  describe('additional validation checks', async () => {
```

Comment:
rather than using pros, I would just indicate that this block of tests is testing the `changedFiles` configuration:

```
describe("changedFiles", () => {});
```

Since we'll be adding _additional_ additional validation checks over time, then I think the assertions within the describe block can describe the algorithm in detail:

```
it("should indicate PR is mergeable if it only modifies the files indicated in changedFiles list");
it('should indicate PR is not-mergeable if it modifies files other than those in the changedFiles list");
```

Reply:
> rather than using pros, I would just indicate that this block of tests is testing the `changedFiles` configuration:
> 
> ```
> describe("changedFiles", () => {});
> ```
> 

I'm a bit confused, because this block tests whether a PR requires additional checks _besides_ the changedFiles and titles checks. (i.e., whether or not we need to check versions, or something else). 

Would it be alright to still call this block "additional validation checks", and further detail each tests' algorithm in the description of the test?